pkg/driver/controller_test.go

```diff
@@ -46,7 +46,9 @@ func TestCreateVolume(t *testing.T) {
 
 	stdVolSize := int64(5 * 1024 * 1024 * 1024)
 	stdCapRange := &csi.CapacityRange{RequiredBytes: stdVolSize}
-	stdParams := map[string]string{}
+	stdParams := map[string]string{
+		"type": cloud.DefaultVolumeType,
```

Comment:
**When volume type is set:**
Request will contain parameter with Volume type attribute as here: https://github.com/Madhan-SWE/ibm-powervs-block-csi-driver/blob/50118644a97d54c19912cadc4ccf67220ac77da0/pkg/driver/controller_test.go#L565

**When volume type is not set:**
Request will contain parameter with no Volume type attribute as here: https://github.com/Madhan-SWE/ibm-powervs-block-csi-driver/blob/50118644a97d54c19912cadc4ccf67220ac77da0/pkg/driver/controller_test.go#L649

Separate test cases already added for various values of volumeType:
`tier1`: https://github.com/Madhan-SWE/ibm-powervs-block-csi-driver/blob/50118644a97d54c19912cadc4ccf67220ac77da0/pkg/driver/controller_test.go#L511
`tier3`: https://github.com/Madhan-SWE/ibm-powervs-block-csi-driver/blob/50118644a97d54c19912cadc4ccf67220ac77da0/pkg/driver/controller_test.go#L556
`notSet`: https://github.com/Madhan-SWE/ibm-powervs-block-csi-driver/blob/50118644a97d54c19912cadc4ccf67220ac77da0/pkg/driver/controller_test.go#L641


Apart from the above test cases, other test cases use a default value for VolumeType because, for other test cases the motive is to test other parameters.